manifests/site.pp

```diff
@@ -72,7 +72,14 @@
     include prometheus::php_fpm
 }
 
-node 'mwtask1.miraheze.org' {
+# new hardware
+node /^mw(1[012][12])\.miraheze\.org$/ {
+    include base
+    include role::mediawiki
+    include prometheus::php_fpm
+}
+
+node 'mwtask(1|101).miraheze.org' {
```

Comment:
This needs to be in regex format: `/^mwtask(1|101)\.miraheze\.org$/`. It could also be `mwtask(1(01)?)` if its better but probably more complicated.